impl/src/main/java/org/ehcache/internal/copy/SerializingCopier.java

```diff
@@ -33,6 +29,10 @@
   private final Serializer<T> serializer;
 
   public SerializingCopier(Serializer<T> serializer) {
+    if (serializer == null) {
+      throw new NullPointerException("A " + SerializingCopier.class + " instance requires a "
```

Comment:
Even though when we read this error message directly from the code it sounds right, once it gets printed in the log it becomes this: "A class org.ehcache.internal.copy.SerializingCopier instance requires a interface org.ehcache.spi.serialization.Serializer instance to copy!"

The words _class_ and _interface_ that got prepended to the FQNs make the error message sound weird. And that's just *my* opinion.